dstk-infra/apollo/src/graphql/model/model.ts

```diff
@@ -1,36 +1,43 @@
-import { objectType } from 'nexus'
-import { Model } from 'objection'
+import { objectType } from 'nexus';
+import { Model } from 'objection';
+import { ObjectionStorageProvider } from '../storage-provider/storageProvider';
 
 export const MLModel = objectType({
     name: 'Model',
     definition(t) {
-        t.id('modelId')
-        t.string('storageProviderId')
-        t.boolean('isArchived')
-        t.string('modelName')
-        t.string('createdBy') // TODO: resolve actual user object
-        t.string('modifiedBy')
-        t.string('dateCreated')
-        t.string('dateModified')
-        t.string('description')
+        t.id('modelId');
+        t.field('storageProvider', {
+            type: 'StorageProvider',
+            async resolve(root, _args, _ctx) {
+                ObjectionStorageProvider.query().findById(root.providerId).first();
+            },
+        });
+        t.boolean('isArchived');
+        t.string('modelName');
+        t.string('createdBy'); // TODO: resolve actual user object
+        t.string('modifiedBy');
+        t.string('dateCreated');
+        t.string('dateModified');
+        t.string('description');
         // holding off on metadata bc I don't want to deal
         // with JSON serialization/deserialization right now
     },
-})
+});
 
 export class ObjectionMLModel extends Model {
-    id!: string
-    storageProviderId!: string
-    isArchived!: boolean
-    modelName!: string
-    createdBy!: string
-    modifiedBy!: string
-    dateCreated!: string
-    dateModified!: string
-    description!: string
+    id!: string;
```

Comment:
I think I goofed and should have used `modelId` here but I'll figure it out one way or the other tomorrow